ui/library/test/utils/pdfWorker.test.ts

```diff
@@ -7,7 +7,7 @@ describe('initPdfWorker', () => {
   it('pdf worker CDN matches the check in pingdom.com', () => {
     initPdfWorker();
     expect(pdfjs.GlobalWorkerOptions.workerSrc).to.equal(
-      '//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.6.347/pdf.worker.min.js'
+      '//cdnjs.cloudflare.com/ajax/libs/pdf.js/2.12.313/pdf.worker.min.js'
```

Comment:
Did you update the check in www.pingdom.com? credentials are in 1password.com. It doesn't block the merge of this PR. make sure you update the uptime check in pingdom.

Reply:
🆗

Reply:
Updated
<img width="665" alt="Screen Shot 2022-04-22 at 09 58 54" src="https://user-images.githubusercontent.com/84034381/164760524-e8fcdef0-0206-4836-b364-cf76fb450696.png">